core/arch/arm/plat-versal2/conf.mk

```diff
@@ -49,6 +49,11 @@ $(call force,CFG_CORE_CLUSTER_SHIFT,1)
 CFG_TZDRAM_START   ?= 0x1800000
 CFG_TZDRAM_SIZE    ?= 0x8000000
 
+# Console selection
+# 0 : UART0[pl011, pl011_0] (default)
+# 1 : UART1[pl011_1]
+CFG_CONSOLE_UART	?= 0
```

Comment:
Prefer only one space before `?=`, there is nothing to align here
`CFG_CONSOLE_UART ?= 0`

Reply:
Done!